pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -984,4 +984,9 @@ self: super: {
     };
   }).override { language-c = self.language-c_0_7_0; };
 
+  # Needs pginit to function and pgrep to verify.
+  tmp-postgres = overrideCabal super.tmp-postgres (drv: {
+    libraryToolDepends = [pkgs.postgresql];
```

Comment:
It's probably not safe to just replace the current value of these attribute with these new ones. Instead, these newbuild tools should be added to the existing lists, if there are any, to make sure that these overrides still work even if upstream changes the definitions in a new version. For example:

    testToolDepends = drv.testToolDepends or [] ++ [pkgs.procps];

Reply:
Thanks, good idea, have fixed it.